src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss

```diff
@@ -0,0 +1,136 @@
+.dialog-container {
+  width: 573px;
+  height: 575px;
+}
+
+.main-content {
+  margin: 15px 0 0 35px;
+}
+
+h4 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  letter-spacing: 0.002em;
+  color: var(--ubs-quaternary-dark-grey);
+}
+
+.service {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--ubs-secondary-grey);
+  margin-top: 25px;
+}
+
+.serviceInput {
+  width: 480px;
+  height: 36px;
+  background: var(--ubs-primary-white);
+  border: 1px solid var(--ubs-grey-blue);
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+textarea {
+  color: var(--ubs-grey-blue);
+  font-size: 14px;
+  resize: none;
+  width: 480px;
+  height: 141px;
+  border: 1px solid var(--ubs-grey-blue);
+  border-radius: 4px;
+}
+
+.service-description {
+  margin-top: 45px;
+
+  h6 {
+    padding-bottom: 20px;
+  }
+}
+
+.row4 {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--ubs-secondary-grey);
+  margin: 30px 30px 15px 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0 8px 0 0;
+
+  input {
+    background: var(--ubs-primary-white);
+    border: 1px solid var(--ubs-grey-blue);
+    border-radius: 4px;
+    height: 36px;
+  }
+}
+
+.input1 {
+  input {
+    width: 72px;
+  }
+}
+
+.input2 {
+  padding-left: 20px;
+
+  input {
+    width: 130px;
+  }
+}
+
+.input3 {
+  padding-left: 5px;
+
+  input {
+    width: 100px;
+  }
+}
+
+.input4 {
+  input {
+    width: 123px;
+  }
+}
```

Comment:
Can we make more appropriate names for classes? 😸

Reply:
done